frontend/packages/container-security/src/components/__tests__/ImageVulnerabilitiesTable.spec.tsx

```diff
@@ -37,7 +37,7 @@ describe('ImageVulnerabilitiesTable', () => {
     const headers = wrapper
       .find(Table)
       .props()
-      .Header();
+      .Header({} as ComponentProps, (str) => str);
```

Comment:
Why do we need `as` type cast here? Aren't all properties in `ComponentProps` optional, so `{}` should be assignable to `ComponentProps` type? :confused:

Reply:
no, all props of `ComponentProps` are required, but for this test, they are not needed

Reply:
OK, thanks for clarification.